scripts/run_experiment.py

```diff
@@ -72,7 +72,11 @@ def run_design(print_lock, running_list, design, work_path, flow_fcn, flow_args)
     try:
         status = flow_fcn(design=design, build_dir=work_path, flow_args=flow_args)
     except BfasstException as e:
-        status = Status(status=e.error, msg=str(e), raise_excep=False)
+        status = (
```

Comment:
The reason this is necessary, instead of just returning part of the exception (eg e.error) is because run_experiment needs to catch the exception and continue to run other designs. run_design, on the other hand, just lets the exception go and it prints to the terminal. This is also the reason behind the two separate if blocks on lines 120 and 127 of bfasst/status.py @jgoeders